RIOT-OS/RIOT
# board/calliope-mini: use openocd as default flasher and rm fscopy

### Contribution description
The `fscopy` flash script is quite hacky and has never been working very well for most people. Also, `openocd` is now nowadays working just fine with the `calliope-mini`. So there is IMHO no reason to stick with the shaky `fscopy` procedure, and as proposed in this PR I even opt for removing that option completely, decreasing maintenance effort...

### Testing procedure
Simply flash the calliope with any example of your choice. The build system should use `openocd` as default option.

### Issues/PRs references
